pkg/clicmd/root_test.go

```diff
@@ -46,7 +46,7 @@ func Test_parseCommand(t *testing.T) {
 	}
 
 	for _, tc := range tests {
-		cmd, err := parseCommand(tc.args)
+		cmd, _, err := parseCommand(tc.args)
```

Comment:
Better to add tests for the new functionality than to ignore it.

Reply:
Updated tests for new parseCommand return vals